docs/FAQ.md

```diff
@@ -506,6 +506,37 @@ Try increasing the
 [`spark.kryoserializer.buffer.max`](https://spark.apache.org/docs/latest/configuration.html#compression-and-serialization)
 from a default of 64M to something larger, for example 512M.
 
+### Why am I getting "Unable to acquire buffer" or "Trying to free an invalid buffer"?
+
+The RAPIDS Accelerator makes use of a tracking system called `RapidsBufferCatalog` for GPU memory
+spill required for out-of-core algorithms, our reliability (retry) framework, and UCX-accelerated shuffle. 
+Spark threads will acquire/free objects from this catalog over time in order to execute a task.
+
+During executor shutdown, we often see race conditions between task threads finishing up and
```

Comment:
I don't think it is often.  This usually happens when the executor is shutting down early because of another problem and the task threads are still running while the executor shuts down. Perhaps something more like

```
When a fatal Spark exception happens the executor process will not wait for task threads to complete before starting to shut down the process.  This shutdown also includes the `RapidsBufferCatalog`. This race can cause the catalog to 
log these messages that can be ignored.
```